BackofficeBundle/Resources/public/coffee/table/tableHeader/TableviewSearchHeader.coffee

```diff
@@ -0,0 +1,27 @@
+TableviewSearchHeader = OrchestraView.extend(
+
+  tagName: "tr"
+  className: "header-search-input"
+
+  initialize: (options) ->
+    @options = options
+    @render()
+    widgetChannel.trigger 'ready', @
+    return
+
+  render: ->
+    for type, column in @options.inputHeader
+        type = "empty" if not tableFieldViewConfigurator[type]?
+        viewClass = tableFieldViewConfigurator[type]
```

Comment:
I think it's not a good idea to add all in the view configurator and the viewconfigurator it's initialize after jquery  so the viewconfigurator so the view is not necessarily available.